### PR TITLE
Store images on the repo

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This is a gallery of [Voici](https://github.com/voila-dashboards/voici) examples
    - `content_path`: the local path to the notebooks, this path will be passed to the `--contents` parameter of `Voici` command, default to `.`
    - `branch`: the branch to be checked out from `repo_url` repository, default to `main`.
    - `dashboard_url`: the URL of the deployed voici dashboard. If the voici dashboard is already deployed elsewhere you can use this field to bypass the build step. If you use this option, you do not need to provide the `repo_url`, `content_path` and the `branch` field.
-   - `image_url`: the URL of the picture to use as thumbnail.
+   - `image_path`: the name of your picture under `public/images`.
```
Comment (trungleduc) [inline README.md]: I think we should support both, users can freely upload their images elsewhere. If they have the file in their repo, it's easy to get a direct link to this image too.

Comment (martinRenou) [reply README.md]: Sounds good 👍🏽

Comment (trungleduc) [reply README.md]: We should restore the deleted line? And maybe precise the `public/images` of the current repo, not the repo of the dashboard.

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks!

**Comment by jtpio** [comment]
Would you mind providing context for storing images on the repo?

**Comment by jtpio** [comment]
> Would you mind providing context for storing images on the repo?

So that if we look at that again in a couple of months / years we understand why.

**Comment by martinRenou** [comment]
> Would you mind providing context for storing images on the repo?

It seems the trick to store images on Github issues is not easily usable anymore and Github is switching to another system. So storing the images ourselves on the server may be easier.

Though in this PR we'll continue supporting externally hosted images.

**Pull request merged**